de-man-3/team-3-project
# added sample dataset for EDA

tested PR creation working!

After cloning project: Verify local repo is tracking origin->this project folder: git remove -v

On Local, when starting work on a new change/feature:
1. Create new local branch: git branch branchname
2. Make your changes. When done, stage them: git add (filenames or .)
3. Commit them to local repo: git commit -m "type a message describing change"
4. Push to remote repo AND create pull request: git push -u origin branchname
5. Go to repo on github website and review pushed commits and click "Create Pull Request".
6. Await review.
